File: PRAK-2/tempCodeRunnerFile.c
#include <stdio.h>
int main()
{
  // test case 1
  int A, B;
  printf("");
  scanf("%d %d", &A, &B);
  // alas
  int alas = (B * B - A * A) / 9;
  printf("Alas = %d cm\n", alas);
  // tinggi
  printf("Tinggi = %d cm\n", A);
  // keliling
  int keliling = A + B + alas;
  printf("Keliling = %d cm\n", keliling);
  // luas
  int luas = 0.5 * alas * A;
  printf("Luas = %d^2 cm", luas);

  // test case 2
  int A1, B1;
  printf("");
  scanf("%d", &A1);
  printf("");
  scanf("%d", &B1);
  // alas
  int alas1 = (B1 * B1 - A1 * A1) / 63;
  printf("Alas = %d cm\n", alas1);
  // Tinggi
  int tinggi1 = A1;
  printf("Tinggi = %d cm\n", tinggi1);
  // keliling
  int keliling1 = A1 + B1 + alas1;
  printf("Keliling = %d cm\n", keliling1);
  // luas
  int luas1 = 0.5 * alas1 * A1;
  printf("Luas = %d^2 cm", luas1);
}

File: PRAK-2/tempCodeRunnerFile.py
#test case 1
pi = 22.0/7.00
jari_jari=float(input(""))
tinggi=float(input(""))

#volume
volume = pi * jari_jari * jari_jari * tinggi
#luas permukaan
luas_permukaan = 2 * pi * jari_jari * (jari_jari + tinggi)
#Menghitung keliling
keliling = 2 * pi * jari_jari

print(f"Volume = {volume:.2f}")
print(f"Luas = {luas_permukaan:.2f}")
print(f"Keliling = {keliling:.2f}")

#test case 2
pi1 = 22.0/7.00
jari_jari1, tinggi1=map(float, input("").split())

#volume
volume1 = pi1 * jari_jari1 * jari_jari1 * tinggi1
#luas permukaan
luas_permukaan1 = 2 * pi1 * jari_jari1 * (jari_jari1 + tinggi1)
#Menghitung keliling
keliling1 = 2 * pi1 * jari_jari1

print(f"Volume = {volume1:.2f}")
print(f"Luas = {luas_permukaan1:.2f}")
print(f"Keliling = {keliling1:.2f}")